kubernetes-python/.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
@swgillespie, since we [encourage](https://github.com/pulumi/pulumi/blob/597a2861b056d4dac1d034abc802b4261078f2bc/cmd/new.go#L587) the use of `virtualenv`, do you think it'd be worth adding `venv/` to `.gitignore` for our Python templates?